tark/tark_web/templates/mane_GRCh37_list.html

```diff
@@ -0,0 +1,93 @@
+{% extends "web_home.html" %}
+
+{% load staticfiles %}
+{% load add_class %}
+{% load add_class %}
```

Comment:
Does `{% load add_class %}` need to be called twice?

Reply:
Hi Bilal, thanks a lot for your review comments. Please find the reply below:
1. as per the documentation, we convert checksum to binary formats for efficient storage as they consume less storage space than string.
2. I have now removed the duplicate load add_class . They were included in the new HTML pages because the duplicate load add_class was also present in the original HTML page from which the new page was derived.

Reply:
I see, thanks @ranjit-s